File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/07_Arrays_Lab/01_Day_Of_Week.php
<?php

$input = intval(readline());

$days = [
  "Monday",
  "Tuesday",
  "Wednesday",
  "Thursday",
  "Friday",
  "Saturday",
  "Sunday"
  ];

if ($input >= 1 && $input <= 7) {
  echo $days[$input - 1];
} else {
  echo "Invalid Day!";
}

?>

File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/07_Arrays_Lab/01_DayOfWeek.cs
using System;

class MainClass {
  public static void Main () {
    
    string[] WeekArr = { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };
    
    int dayNumber = int.Parse(Console.ReadLine());

    if (dayNumber - 1 >= 0 && dayNumber - 1 < WeekArr.Length) {
      Console.WriteLine(WeekArr[dayNumber - 1]);
    } else {
      Console.WriteLine("Invalid Day!");
    }

  }
}